### ISSUE TITLE
Identity Icon(s) are not showing in Firefox 45.0a1 (Nightly)

### ISSUE THREAD

**Issue by johngraciliano** [johngraciliano/simplewhite#15]
Changes in **Firefox** are so that _identity icons_ are not show in **Firefox _Nightly_ 45.0a1 (2015-11-13)**.  The _information panel_ remains available (it will pop up when pressing where the icons should be).  It does not seem to affect functionality, however this is a **_very severe issue**_ and takes top priority.

**Comment by johngraciliano** [comment]
While working on this issue I noticed a change to the messages in the _information panel_ that also affects **Firefox _Developer Edition_ 44.0a2**.  In particular, I noticed **Firefox**'s  _about:_ pages (e.g., the home page or the add-ons page) incorrectly report _"Your login could be compromised"_. Correcting this takes precedence because it affects a version closer to general release.
I must make sure there are no other similar problems.
This is related to **Firefox** bug 1221771, which still open as I add this to the commentary.
